README.md

```diff
@@ -1,29 +1,49 @@
 # health-check-slack 🤖
+
 Simple Node app to do health checks on 1 or more urls and send notifications to a slack channel
 
 ## Install :star2::package:
-git clone https://github.com/nclr/health-check-slack.git  
-cd health-check-slack  
-npm install  
+
+git clone https://github.com/profusion/health-check-slack.git
```

Comment:
please do this one in a new commit as the original repo is unlike to pick this part :-) or just omit it